lib/openid_token_proxy/token/refresh.rb

```diff
@@ -17,6 +17,11 @@ def require_valid_token
           )
           response.headers['X-Token'] = current_token.access_token
           response.headers['X-Refresh-Token'] = current_token.refresh_token
+
+          instance_exec(
+            current_token.access_token,
```

Comment:
It would probably be a good idea to keep the block signature similar to the [`token_acquirement_hook`](https://github.com/hyperoslo/openid-token-proxy/blob/master/lib/openid_token_proxy/concerns/callback_controller.rb#L21), passing in the token object, rather than just the access token string.